src/background/sidebar-injector.js

```diff
@@ -262,10 +266,8 @@ export function SidebarInjector() {
         "Hypothesis extension can't be used on Hypothesis LMS assignments"
       );
     } else {
-      await injectConfig(tab.id, config);
-
       const result = /** @type {{ installedURL: string }|null} */ (
-        await injectIntoHTML(tab)
+        await injectIntoHTML(tab, config)
```

Comment:
I have moved the call to `injectConfig` inside `injectIntoHTML`, to make it consistent with `injectIntoVitalSourceReader`.